bin/helpers/capabilityHelper.js

```diff
@@ -147,14 +149,27 @@ const caps = (bsConfig, zip) => {
   })
 }
 const getAccessibilityPlatforms = (bsConfig) => {
-  const browserList = bsConfig.browsers;
+  const browserList = [];
+  if (bsConfig.browsers) {
+    bsConfig.browsers.forEach((element) => {
+      element.versions.forEach((version) => {
+        browserList.push({...element, version, platform: element.os + "-" + element.browser});
+      });
+    });
+  }
+  
   const accessibilityPlatforms = Array(browserList.length).fill(false);
   let rootLevelAccessibility = false;
   if (!Utils.isUndefined(bsConfig.run_settings.accessibility)) {
-    rootLevelAccessibility = bsConfig.run_settings.accessibility.toString() === 'true'
+    rootLevelAccessibility = bsConfig.run_settings.accessibility.toString() === 'true';
   }
   browserList.forEach((browserDetails, idx) => {
-    accessibilityPlatforms[idx] = (browserDetails.accessibility === undefined) ? rootLevelAccessibility : browserDetails.accessibility
+    accessibilityPlatforms[idx] = (browserDetails.accessibility === undefined) ? rootLevelAccessibility : browserDetails.accessibility;
+    if (browserDetails.browser && browserDetails.browser.toLowerCase() !== 'chrome') {
+      logger.warn(`Accessibility Automation will run only on Chrome browsers for ${browserDetails.platform}.`);
+    } else if (browserDetails.version && (!browserDetails.version.includes('latest') || browserDetails.version < 94)) {
```

Comment:
@07souravkunda `!browserDetails.version.includes('latest')` it should not have not condition currently its throwing error when we provide any other version like 118, 120